src/Tizen.NUI.Scene3D/src/public/Controls/Panel.cs

```diff
@@ -0,0 +1,377 @@
+            this.PositionUsesPivotPoint = true;
+        }
+
+        /// <summary>
+        /// Resolution of the Panel.
+        /// The resolution is independent from the Panel size property.
+        /// The resolution defines a plane that the 2D UI scene will be rendered.
+        /// And the shape of the panel plane is defined by aspect ratio of the input resolution.
+        /// The plane is cleared by white color.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector2 PanelResolution
+        {
+            get
+            {
+                return GetPanelResolution();
+            }
+            set
+            {
+                SetPanelResolution(value);
+            }
+        }
+
+        /// <summary>
+        /// Root View of 2D UI content.
+        /// The content is rendered on the plane of the Panel by using off screen rendering.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View Content
+        {
+            get
+            {
+                return GetContent();
+            }
+            set
+            {
+                SetContent(value);
+            }
+        }
+
+        /// <summary>
+        /// Whether Transparent background is used or not. Default value is false
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool Transparent
+        {
+            get
+            {
+                bool transparent = false;
+                Tizen.NUI.PropertyValue pValue = GetProperty(Interop.Panel.PropertyTransparentIndexGet());
+                pValue.Get(out transparent);
+                pValue.Dispose();
+                return transparent;
+            }
+            set
+            {
+                var transparent = new Tizen.NUI.PropertyValue(value);
+                SetProperty(Interop.Panel.PropertyTransparentIndexGet(), transparent);
+                transparent.Dispose();
+            }
+        }
+
+        /// <summary>
+        /// Whether the content is rendered as double sided or not.
+        /// Default value is false.
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool DoubleSided
+        {
+            get
+            {
+                bool doubleSided = false;
+                Tizen.NUI.PropertyValue pValue = GetProperty(Interop.Panel.PropertyDoubleSidedIndexGet());
+                pValue.Get(out doubleSided);
```

Comment:
```suggestion
                pValue?.Get(out doubleSided);
```
Please check pValue is null.

Hear, and all other codes what you create

Reply:
Thank you.